alerting/src/entities/alerts.entity.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { GeoJSON } from 'geojson';
+
+type WCSConfig = {
+  scale?: number;
+  offset?: number;
+  pixelResolution?: number;
+};
+
+export type AlertConfig = {
+  type: string;
+  serverLayerName: string;
+  baseUrl: string;
+  wcsConfig: WCSConfig;
+};
+
+@Entity()
+export class Alert {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  email: string;
+
+  @Column()
+  prismUrl: string;
+
+  @Column({ nullable: true })
+  alertName?: string;
+
+  // Configuration of the type
+  // TODO - Use type from frontend
```

Comment:
Maybe we make a proper yarn-workspaces setup with a `alerting` package, `frontend` package and `common` package to hold this type?

Reply:
Yes I think that's the next step!